test/api.rdoctest

```diff
@@ -47,44 +44,19 @@ The main feature of HoboFields, aside from the migration generator, is the abili
 
 Field values are returned as the type you specify.
 
-        >> a = Advert.new :body => "This is the body"
+        >> a = Advert.new :body => "This is the body", id: 1, title: "title"
         >> a.body.class
-        => HoboFields::Types::Text
+        => String
```

Comment:
I was expecting this to be Text per my comment above.

Reply:
Responded in the above comment. Resolving this to single thread the conversation.